### PR TITLE
Paraxxius in Antorus.

### PR DESCRIPTION
**Pull request by ghost**
Add Paraxxius GO ship spawn, Add Paraxxius Inquisitor spawn, Add SAI for Paraxxius Inquisitor.

**Changes proposed:**

-  Add spawn Go ship Paraxxius (We need more precise location, more accurate id, a more accurate display id.
I took a similar basis.)
-  Need focusing crystal with cosmetic auras.
-  

**Issues addressed:** Closes #  (insert issue tracker number)

**Tests performed:** (Does it build, tested in-game, etc.)

**Known issues and TODO list:** (add/remove lines as needed)

- [ ] 
- [ ]

### CODE REVIEW

File: sql/ashamane/world/2018_08_09_85_world_antorus.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+(125429,0,3,0,0,0,100,0,11000,11000,32000,32000,,11,249126,0,0,0,0,0,5,0,0,0,0,0,0,0,'Paraxis Inquisitor - In Combat - Cast Mind Blast on Random Hostile Target'),
+(125429,0,4,0,9,0,100,0,5,30,5000,5000,,11,249155,0,0,0,0,0,5,0,0,0,0,0,0,0,'Paraxis Inquisitor - At Range 30 Yards - Cast Torment');
+
+DELETE FROM `gameobject_template` WHERE `entry` = 500000;
+INSERT INTO `gameobject_template` (`entry`, `type`, `displayId`, `name`, `IconName`, `castBarCaption`, `unk1`, `size`, `Data0`, `Data1`, `Data2`, `Data3`, `Data4`, `Data5`, `Data6`, `Data7`, `Data8`, `Data9`, `Data10`, `Data11`, `Data12`, `Data13`, `Data14`, `Data15`, `Data16`, `Data17`, `Data18`, `Data19`, `Data20`, `Data21`, `Data22`, `Data23`, `Data24`, `Data25`, `Data26`, `Data27`, `Data28`, `Data29`, `Data30`, `Data31`, `Data32`, `RequiredLevel`, `AIName`, `ScriptName`, `VerifiedBuild`) VALUES
+('500000','43','29031','Paraxxius','Paraxxius','','0','10','-1','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','SmartAI','','0');
```
Comment (Traesh) [inline sql/ashamane/world/2018_08_09_85_world_antorus.sql:13]: Why entry 500000 ? Don't you have sniff of that gameobject ?

Comment (ghost) [reply sql/ashamane/world/2018_08_09_85_world_antorus.sql:13]: Sorry, i don't have any sniff. Reason for custom entry.
Search in Gameobjects.db2 also don't have result.

Comment (leviathonbeast) [reply sql/ashamane/world/2018_08_09_85_world_antorus.sql:13]: i consider this an invalid fix until you get sniff data for it.

**Comment by Traesh** [comment]
I close it for now, we'll re-open with proper entry

**Comment by Lartty96** [comment]
Try with 267004

**Pull request closed without merging**